addons/test_discuss_full/tests/test_performance.py

```diff
@@ -71,10 +71,11 @@ class TestDiscussFullPerformance(HttpCase, MailCommon):
     #           - search ir_attachment (_compute_avatar_128)
     #           - count discuss_channel_member (member_count)
     #           - _compute_message_needaction
+    #           - _compute_is_channel_owner_admin
     #           - fetch res_groups (group_public_id)
     #           - fetch ir_module_category (group_public_id)
     #           - search group_ids
-    _query_count_init_messaging = 35
+    _query_count_init_messaging = 36
```

Comment:
To double check but we shouldn't need more queries if we are just reading an extra field on the member table.
Do we need an extra query because of the groups on the field and the sudo?